### PR TITLE
Update UG edit command

### LINKED ISSUE #138: Update edit command in UG for CS2101

### PR DESCRIPTION
**Pull request by DennieDan**
Closes #138

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -108,15 +108,26 @@ Format: `list`
...
+**Format:** `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [f/FEE] [paid/PAID_AMOUNT] [owed/OWED_AMOUNT]`
+
+**Examples:**
+
+`edit 1 p/87438808 e/alexyeoh100@example.com` Edits the phone number and email address of the 1st student to be `87438808` and `alexyeoh100@examnple.com` respectively.
+
+`edit 2 paid/1200.00 owed/0` Edits the paid amount of the 2nd student to be `$1200.00` and edits the owed amount to be `$0.00`.
+![editResult.png](images/editResult.png)
+
+<box type="important" header="##### Constraints">
```
Comment (Feng1231) [inline docs/UserGuide.md:120]: There is an additional # here

File: docs/UserGuide.md [hunk 2]
```diff
@@ -108,15 +108,26 @@ Format: `list`
 
 Edits an existing student in the address book.
 
-Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] …​`
+**Format:** `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [f/FEE] [paid/PAID_AMOUNT] [owed/OWED_AMOUNT]`
```
Comment (Soumyadip-cmd) [inline docs/UserGuide.md:111]: I think we changed [f/FEE] to [r/RATE]

File: docs/UserGuide.md [hunk 3]
```diff
@@ -108,15 +108,26 @@ Format: `list`
...
+
+**Examples:**
+
+`edit 1 p/87438808 e/alexyeoh100@example.com` Edits the phone number and email address of the 1st student to be `87438808` and `alexyeoh100@examnple.com` respectively.
```
Comment (Soumyadip-cmd) [inline docs/UserGuide.md:115]: Could perhaps add <br> between the example and the explanation

File: docs/UserGuide.md (hunk 1 above)
Comment (Soumyadip-cmd) [inline docs/UserGuide.md:120]: Might be better to use 4 # to standardise size of header

**Review by Soumyadip-cmd — COMMENTED** [review]
Just minor changes

**Review by Soumyadip-cmd — APPROVED** [review]
LGTM

**Pull request merged**